crowbar_framework/app/assets/javascripts/application.js

```diff
@@ -85,7 +85,7 @@ jQuery(document).ready(function($) {
     ]
   });
 
-  $('[data-blockui]').live('submit', function(event) {
+  $('[data-blockui]').live('click submit', function(event) {
```

Comment:
Is this correct? Maybe I have something wrong (I only have part of the patch applied), but I'm getting blocking UI for every submit, e.g. when I'm editing barclamp proposal